honeybee/aperturetype.py

```diff
@@ -5,15 +5,25 @@
 class _ApertureType(object):
     __slots__ = ()
 
+    def __init__(self):
+        pass
+
+    @classmethod
+    def from_dict(cls, data):
+        """Initialize ApertureType from a dictionary.
+
+        Args:
+            data: A dictionary representaion of the aperture type.
+        """
+        return cls()
```

Comment:
That one is actually how it's implemented for any aperture type that doesn't have any extra properties on it.  An example of an extra property would be like storing the `fraction_operable` on the `OperableWindow` aperture type.  For such a type with extra properties, we can overwrite this base `from_dict` method such that it correctly sets the `fraction_operable`. So we basically need this simple base method so that we can call `from_dict` for any aperture type, whether it has extra properties or not.

Reply:
This issue no longer exists now that we decided face types and aperture types will just be text strings in the schema.